From: Antoine Musso <hashar@free.fr>
Date: Fri, 22 Jul 2016 15:35:07 +0200
Subject: WMF: constraint apscheduler to <3.1.0

apscheduler 3.1.0 fail with old setuptools
https://github.com/agronholm/apscheduler/issues/123

That is solved in 3.2.x apparently.

Change-Id: I9a736379f9391a91219afa1e9e192e53ca8883fe
---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 793773e..f98bdd6 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -10,7 +10,7 @@ extras
 statsd>=1.0.0,<3.0
 voluptuous>=0.7
 gear>=0.5.7,<1.0.0
-apscheduler>=3.0
+apscheduler>=3.0,<3.1.0
 PrettyTable>=0.6,<0.8
 babel>=1.0
 six>=1.6.0
